product-features/farming/how-to.md

```diff
@@ -0,0 +1,208 @@
+
+### Providing liquidity to get LP tokens
+
+Now that you've chosen a farm to stake in, you'll need LP tokens, which are obtained by adding liquidity to a pool.
+
+1. Click on the Pool tab in the top navigation bar.
+
+<figure><img src="../../.gitbook/assets/liquidity-providers/adding-liquidity-1-pool-tab.png" alt="" width="375"><figcaption></figcaption></figure>
+
+2. Find the pool that matches the **Trading Pair** you noted before. This is the liquidity pool linked to the farm you've selected.&#x20;
+3. Click on the pool. This will open the pool control panel, where you can add liquidity.&#x20;
+
+We have a [Guide to Adding Liquidity](../liquidity-providers/how-to.md#adding-liquidity) that you can follow to obtain LP tokens.
+
+## :sunflower: How to put LP tokens in a farm (stake)
+
+If you have LP tokens, you're ready to start staking them in a farm and earning rewards!
+
+### Step 1
+
+Go to the [Farms page](https://app.alexlab.co/farm) and locate your farm of interest. You can access it by navigating to [https://app.alexlab.co/](https://app.alexlab.co/) and selecting the Earn -> Farm tab.&#x20;
+
+At the top of the farm list, you'll see the farms suggested by the ALEX Lab Platform based on your LP tokens balance. In the example below, the suggested farm is STX-ALEX. This indicated that the user has provided liquidity in the STX-ALEX pool and has LP tokens available for farming.
+
+<figure><img src="../../.gitbook/assets/farming/suggested-farms.png" alt=""><figcaption><p>Example of farm suggestions. This user is a STX-ALEX pool provider and possesses STX-ALEX LP tokens that can be staked in the STX-ALEX farm.</p></figcaption></figure>
+
+### Step 2
+
+Select the farm you want to stake in from the farm list.
+
+<figure><img src="../../.gitbook/assets/farming/select-farm.png" alt=""><figcaption></figcaption></figure>
+
+{% hint style="warning" %}
+When hovering over a farm, you may notice a "+ Stake LP" button. This serves as a visual indicator for the selected farm. Clicking it will take you to the same screen as clicking anywhere on the farm's row.
+{% endhint %}
+
+### Step 3
+
+Once you have selected the farm, enter the amount of LP tokens you would like to stake, or click "Max" to use all available LP Tokens.&#x20;
+
+Next, choose the number of reward cycles you want to lock your tokens into the farm. Each cycle is approximately 3.5 days. 
+
+<figure><img src="../../.gitbook/assets/farming/add-lp-staking.png" alt="" width="375"><figcaption><p>Example with an amount of LP tokens to stake for 32 cycles.</p></figcaption></figure>
+
+{% hint style="info" %}
+Your staked amount will start generating yield from the next upcoming cycle, as the current cycle is in "cooldown" period. To maximize the APR you earn, it's best to stake for longer cycle periods to avoid missing out on any reward cycles due to this cooldown cycle. That's why 32-cycle staking is recommended.
```

Comment:
"That's why 32-cycle staking is recommended."

"That's why 32-cycle staking is recommended if your goal is to maximize earnings."

Reply:
Corrected